cspell-config/cspell-dev.txt

```diff
@@ -110,3 +110,9 @@ zkout
 zksolc
 zkvyper
 retryable
+itsacoyote
```

Comment:
This seems like a strange addition... where is `itsacoyote` showing up?

Reply:
Same -- temporary, it's mentioned in todo in some comment.